server/api/follower_handler.py

```diff
@@ -0,0 +1,66 @@
+from flask import jsonify, Blueprint
+from models.user import User
+from database import db
+from api.auth_handler import token_getter
+
+follower_handler = Blueprint('follower_handler', __name__)
+
+
+@follower_handler.route('/followers/<user_id>', methods=['GET'])
+def getFollowers(user_id):
+    auth_token = token_getter()
```

Comment:
Any reason why you guys don't use the login_required decorator and instead call token_getter in each request?